CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+# Contributing
+
+## How to submit changes or bug reports
+Submit an issue or bug with reproduction steps in the issue tag
+
+## Guidelines for contributions
+Tests must still pass or have good explanation for change. Only additions to the tests and improvements to test coverage are encouraged.
+
+## Setting up a development environment
+`go get` or `go install` puts sqlvet in `$GOPATH` or `$HOME/go`. You're able to fork and the repo into another directory to make changes. `go test ./...` works pretty well for testing changes.
+
+## Development Workflow Requirements
+The `const version` needs to be updated with changes before new releases are tagged and published.
```

Comment:
for future improvements, we could also dynamically inject the version value at build time during release CI runs using the tag value so we don't need to maintain versions in multiple places.